README.md

```diff
@@ -20,3 +20,7 @@ There is currently no official installer for Earth Enterprise Fusion & Server.
 
 ### Known Issues
 Known issues can be tracked via [wiki page](https://github.com/google/earthenterprise/wiki/Known-Issues), which is updated weekly.
+
```

Comment:
Most projects have badges under the title in the readme. 
https://github.com/nodejs/node
https://github.com/facebook/prepack
https://github.com/tensorflow/tensorflow